docs/content/components/form/checkbox/checkbox.mdx

```diff
@@ -36,15 +30,17 @@ The checkbox component is comprised of a checkbox label and a checkbox input. If
 
 ## Usage
 
+Checkboxes allow user to to toggle a single option or to select multiple options from a list. Each checkbox works independently, so selecting one does not affect the others. Users can choose none, one, or many items.
```

Comment:
The following section also started with "Checkboxes" maybe you find another start?